config/config.yml

```diff
@@ -479,3 +479,7 @@ PERMITTED_HOSTS: [
 ]
 
 USER_RENAME_LIMIT_DAYS: 7
+
+# Number of seconds to retain the original creator for works that have been
```

Comment:
The issue says 72 hours, what is the units ?

Reply:
Whoops, should be hours. I've updated the name to make that clear